test/testcafe/spec/Identify_spec.js

```diff
@@ -103,7 +103,7 @@ test.requestHooks(identifyRequestLogger, identifyMock)('should be able to submit
   });
   await t.expect(req.method).eql('post');
   await t.expect(req.url).eql('http://localhost:3000/idp/idx/identify');
-  await t.expect(reqHeaders['x-device-fingerprint']).notOk();
+  // await t.expect(reqHeaders['x-device-fingerprint']).notOk();
```

Comment:
is it actually failing? (add todo comment?)

Reply:
yeah this is now a wrong check because now re-rendering of widget happens after the error. I can add a comment.

Reply:
This should continue to operate as before, as the feature to compute the device fingerprint is turned off. What is this returning?

Reply:
the headers gets this 
```x-device-fingerprint: mock-device-fingerprint```

so it turn out to be truthy instead of `falsy`
does turning off the feature also means that it won't anything to headers ?

Reply:
fixed it. my widgetrc had it to true.. Thanks a ton @jmelberg-okta